src/_guides/language/sound-problems.md

```diff
@@ -455,7 +456,7 @@ in [Effective Dart](/guides/language/effective-dart/).
 
 Strong mode currently disallows overriding a field with another field or with a getter/setter. Generally speaking making the superclass field into a getter/setter pair is a simple solution.  Note that we do allow an interface which declares a field to be implemented with a field, or with a getter/setter. Overriding a field with another field or with a getter/setter pair generally leaves the old field around using up memory in the object. It’s technically still possible to access the field, but most code simply leaves it around unused.
```

Comment:
Is this in a comment?  Both strong mode and DDC now allow/support field overrides.